### ISSUE TITLE
ONE-Wallet should be accessible through testnet

### ISSUE THREAD

**Issue by DylanCkawalec** [harmony-one/chrome-extension-wallet#101]
There's been issues surfacing in the developer community about the accessibility of the 1Wallet and it's use in testnet. In order for wallet users to deploy a contract, they have to test their application on mainnet since the test net does not work. 

When funds are distributed to the 1wallet, there's been additional mainnet issues where users are loosing their funds while testing the application. This needs to be addresses in details. 

Any user who has found such an issue should report their concerns under this thread as soon as possible.

**Comment by lijiang2087** [comment]
this has been resolved.
